transcom/mymove
# B 21773 int

## [B-21773t](https://www13.v1host.com/USTRANSCOM38/TeamRoom.mvc/Show/808731)

## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test
- must have "counselor_move_create" FF on
1. Access the Office app
2. Login as a services counselor
3. go to customer search- type some letters to get the "add customer" button to show up
4. Clcik Add Customer
5. validate the "Is this a Bluebark Move?" shows up
6. When yes is selected - the okta account and Non-Cac users forms are hidden
7.  When "safety_move_flag" is on and the user has safety Privs- validate that you can not have safety move and bluebark able to both be "Yes"
8. fill out the customer info submit
9. Validate on "Tell us about the orders" page that the bluebark orders type is prefilled and unchangeable
10. go back and select a diff order type- make sure that blueback can not be selected on this page 
11. 


### Frontend


### Backend



## Screenshots
![image](https://github.com/user-attachments/assets/718a15cb-bd86-4275-a8c2-98fe16f1b016)
![image](https://github.com/user-attachments/assets/f55cf1ad-6905-44ad-a766-7763ebeed8e1)

![image](https://github.com/user-attachments/assets/3ab9f798-bfaa-49fe-8565-73ea23db717b)
![image](https://github.com/user-attachments/assets/0c7d676e-057b-414e-98c2-6706503668d1)
![image](https://github.com/user-attachments/assets/9d6251c6-56c5-4c5e-952e-449bc871d952)
